pyan/writers.py

```diff
@@ -234,127 +208,115 @@ def run(self):
-        self.write('</y:GroupNode>')
+        self.write("</y:GroupNode>")
         self.dedent()
-        self.write('</y:Realizers>')
+        self.write("</y:Realizers>")
         self.dedent()
-        self.write('</y:ProxyAutoBoundsNode>')
+        self.write("</y:ProxyAutoBoundsNode>")
         self.dedent()
-        self.write('</data>')
+        self.write("</data>")
         self.write('<graph edgedefault="directed" id="%s::">' % graph.id)
         self.indent()
 
     def finish_subgraph(self, graph):
-        self.log('Finish subgraph %s' % graph.label)
+        self.log("Finish subgraph %s" % graph.label)
         self.dedent()
-        self.write('</graph>')
+        self.write("</graph>")
         self.dedent()
-        self.write('</node>')
+        self.write("</node>")
 
     def write_node(self, node):
-        self.log('Write node %s' % node.label)
+        self.log("Write node %s" % node.label)
         width = 20 + 10 * len(node.label)
         self.write('<node id="%s">' % node.id)
         self.indent()
         self.write('<data key="d0">')
         self.indent()
-        self.write('<y:ShapeNode>')
+        self.write("<y:ShapeNode>")
         self.indent()
         self.write('<y:Geometry height="%s" width="%s"/>' % ("30", width))
-        self.write('<y:Fill color="%s" transparent="false"/>'
-                   % node.fill_color)
-        self.write('<y:BorderStyle color="#000000" type="line" '
-                   'width="1.0"/>')
-        self.write('<y:NodeLabel>%s</y:NodeLabel>'
-                   % node.label)
+        self.write('<y:Fill color="%s" transparent="false"/>' % node.fill_color)
+        self.write('<y:BorderStyle color="#000000" type="line" ' 'width="1.0"/>')
+        self.write("<y:NodeLabel>%s</y:NodeLabel>" % node.label)
         self.write('<y:Shape type="ellipse"/>')
         self.dedent()
-        self.write('</y:ShapeNode>')
+        self.write("</y:ShapeNode>")
         self.dedent()
-        self.write('</data>')
+        self.write("</data>")
         self.dedent()
-        self.write('</node>')
+        self.write("</node>")
 
     def write_edge(self, edge):
         self.edge_id += 1
         source = edge.source
         target = edge.target
-        self.write(
-                '<edge id="%s" source="%s" target="%s">'
-                % (self.edge_id, source.id, target.id))
+        self.write('<edge id="%s" source="%s" target="%s">' % (self.edge_id, source.id, target.id))
         self.indent()
         self.write('<data key="d1">')
         self.indent()
-        self.write('<y:PolyLineEdge>')
+        self.write("<y:PolyLineEdge>")
         self.indent()
-        if edge.flavor == 'defines':
-            self.write('<y:LineStyle color="%s" '
-                       'type="dashed" width="1.0"/>'
-                       % edge.color)
+        if edge.flavor == "defines":
+            self.write('<y:LineStyle color="%s" ' 'type="dashed" width="1.0"/>' % edge.color)
```

Comment:
Auto-format borkage. Maybe fix all by searching for `' '`, and then a manually curated replace with an empty string? (Need to curate it, because `' '` could be just an empty string, which could somewhere be the right thing.)